qiita_core/configuration_manager.py

```diff
@@ -234,6 +240,14 @@ def _get_main(self, config):
             self.key_file = join(install_dir, 'qiita_core', 'support_files',
                                  'ci_server.key')
 
+        self.help_email = config.get('main', 'HELP_EMAIL')
+        if not self.help_email:
+            self.help_email = 'qiita.help@gmail.com'
+
+        self.sysadmin_email = config.get('main', 'SYSADMIN_EMAIL')
+        if not self.sysadmin_email:
+            self.sysadmin_email = 'jdereus@health.ucsd.edu'
```

Comment:
I'm on the fence about this being better than actually raising an error if those are not defined; what do you think?

Reply:
agree! Thus, we remove "personalized" emails completely. What type of error do you suggest?

Reply:
I guess something like: `raise ValueError('You need to have a value in the HELP_EMAIL to continue')` but anything that lets the person configuring/starting the system that they are missing that value should work fine. Thank you.

Reply:
Done. I was wondering about the class of the error, not the message ;-) Sorry for being unspecific.